servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.cpp

```diff
@@ -2364,8 +2365,7 @@ void RenderForwardClustered::_render_shadow_append(RID p_framebuffer, const Page
 	_fill_instance_data(RENDER_LIST_SECONDARY, p_render_info ? p_render_info->info[RS::VIEWPORT_RENDER_INFO_TYPE_SHADOW] : (int *)nullptr, render_list_from, render_list_size, false);
 
 	{
-		//regular forward for now
-		bool flip_cull = p_use_dp_flip;
+		bool flip_cull = p_use_dp_flip || p_reverse_cull_face;
```

Comment:
I think this will break the logic of p_use_dp_flip as the value of flip_cull gets overriden. In the case that p_reverse_cull_face is true, p_use_dp_flip is essentially ignored.

Reply:
Make sure to test with an OmniLight using the "Dual Paraboloid" shadow mode